pom.xml

```diff
@@ -77,6 +79,7 @@
     <powermock.version>2.0.2</powermock.version>
     <mockito.version>3.7.7</mockito.version>
     <hamcrest.version>1.3</hamcrest.version>
+    <junit.version>4.12</junit.version>
```

Comment:
`4.13.2` would be the latest junit4 version which contains a lot of bugfix.

Reply:
ok